### PR TITLE
Feature/fix css leaderboard

### PR DESCRIPTION
**Pull request by kubes2020**
# Description

Fixes #

- What work was done? Fix header spacing for Leaderboard and shorten the text for sign in
- Why was this work done? Spacing was off, the sign in was verbose
- What feature / user story is it for? N/A
- Any relevant links or images / screenshots N/A

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/TrophyRoom/Leaderboard.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const Leaderboard = () => {
   useEffect(() => {
     //Getting data from backend for leaderboard
     getLeaderboard(authState).then(res => {
-      console.log(res);
+      console.log('res from getLeaderBoard', res);
```
Comment (tompsherman) [inline src/components/pages/TrophyRoom/Leaderboard.js]: we should remove console logs before pushing to [main] if possible.

not to be a stickler, but Ryan Hamblin destroyed my code from last labs in the guided project this week bc i'd left in so many console.logs lol. it was mortifying!

**Review by tompsherman — APPROVED** [review]
looks good to me, def was a little weird with the w/l stacked on a win & loss heading

File: src/components/pages/TrophyRoom/Leaderboard.js (hunk 1 above)
Comment (kubes2020) [reply src/components/pages/TrophyRoom/Leaderboard.js]: Good catch! I totally forgot to look for those.

**Review by AprilAshby — APPROVED** [review]
This looks great. Can't wait to see it in action!

**Pull request merged**